### ISSUE TITLE
Redirect after login to custom icon

### ISSUE THREAD

**Issue by Biepa** [BookStackApp/BookStack#1491]
**Describe the bug**
Often (unfortunately not every time) when I login, instead of getting to the overview I get redirected to the link, where my custom icon (set in settings) is located. So the only thing i see, is the icon.
(Example URL I get redirected to: https://domain.com/uploads/images/system/2018-01-Sep/scaled-200-43/image.png

**Steps To Reproduce**
Don't know hot reproduce it in the moment.
But get it like 50% of the times i login. (also on mobile)

**Expected behavior**
Login should redirect to some kind of set overview page.

**Your Configuration (please complete the following information):**
 - Exact BookStack Version (Found in settings): v0.26.2
 - Hosting Method (Nginx/Apache/Docker): linuxserver.io docker image

**Comment by ssddanbrown** [comment]
Thanks for reporting @Biepa,
Am I right in thinking you're using the local_secure image storage option?

**Comment by Biepa** [comment]
That's right.

Am 14. Juni 2019 15:55:40 MESZ schrieb Dan Brown <notifications@github.com>:

**Comment by chrissly90** [comment]
I have the same Problem, ![looks like this](https://drop.tinfoil-hat.net/file/s6IUYWquUw9qkdER/u7mmBfAIzPmkSIea/Bug1.png) for me

**Comment by ssddanbrown** [comment]
@chrissly90 I think that may be a different issue.
Can you check what git branch you're on? You should be on `release`. It looks like the styles have not loaded for you which are hosted on `master`. Otherwise, Check if you have set the `APP_URL` option in your `.env` file.

**Comment by chrissly90** [comment]
> @chrissly90 I think that may be a different issue.
> Can you check what git branch you're on? You should be on `release`. It looks like the styles have not loaded for you which are hosted on `master`. Otherwise, Check if you have set the `APP_URL` option in your `.env` file.

You were right, had nothing to do with this issue and editing the .env file got me further

**Comment by ssddanbrown** [comment]
Just come across this while looking over issues. We've made some tweaks here and there since this was raised  so hoping this should not really be a problem anymore. If still getting this then re-uploading the custom logo should help as it'll be added to the correct storage location (In a public space) to be accessible. Going to close but feel free to comment or open a new issue, referencing this one, if still problematic.

**Comment by Sys-Monitor** [comment]
We started to get the same issue after migrating to local_secure so this is still a thing. After trying to reupload I was met with "An error has occured" which was resolved by copying over the image from storage/uploads/images (since that's what we did when we migrated as instructed, so we technically had a backup).
It doesn't seem to like deleting the images in the old public/uploads/images location when migrating. At least not the things that should be public like the app icon.
